lib/common/zstd_internal.h

```diff
@@ -379,10 +385,8 @@ MEM_STATIC U32 ZSTD_highbit32(U32 val)   /* compress, dictBuilder, decodeCorpus
 }
 
 /**
- * Computes CTZ on a U64.
- * This will be slow on 32-bit mode, and on unsupported compilers.
- * If you need this function to be fast (because it is hot) expand
- * support.
```

Comment:
I see that this comment is a bit confusing. Its maybe a bit out of date, because I changed it to a `size_t` instead of a `U64`, so the slowness on 32-bit mode should be resolved. Maybe change it to:

```c
/**
 * Counts the number of trailing zeros of a `size_t`.
 * Most compilers should support CTZ as a builtin. A backup
 * implementation is provided if the builtin isn't supported, but
 * it may not be terribly efficient.
 */
```